packages/template-retail-react-app/app/components/login/index.jsx

```diff
@@ -77,6 +79,25 @@ const LoginForm = ({
                                 id="login_form.action.create_account"
                             />
                         </Button>
+                        <SocialLogin idps={idps} />
+
+                        <Stack direction="row" spacing={1} justify="center">
+                            <Text fontSize="sm">
+                                <FormattedMessage
+                                    defaultMessage="Don't have an account?"
+                                    id="login_form.message.dont_have_account"
+                                />
+                            </Text>
+                            <Button variant="link" size="sm" onClick={clickCreateAccount}>
+                                <FormattedMessage
+                                    defaultMessage="Create account"
+                                    id="login_form.action.create_account"
+                                />
+                            </Button>
+                        </Stack>
+                        <Button variant="link" size="sm" onClick={clickPasswordlessLogin}>
+                            <FormattedMessage defaultMessage="Passwordless Login" />
+                        </Button>
```

Comment:
now that i've merged the passwordless login UI buttons. Should we update this to have the `clickPasswordlessLogin` function triggered for the `Continue Securely` button?